teams/internal-projects.yaml

```diff
@@ -0,0 +1,41 @@
+# ------------------------------------------------------------------------------
+# Administrative OpenRail projects
+# ------------------------------------------------------------------------------
+Administrative Projects:
+  description: Maintainers of OpenRail projects that are not railway-specific
```

Comment:
"not railway-specific" might not be the best distinction here, as they are in some way still railway specific but not directly used in the railway business. Maybe "Maintainers of repos used for managing the technical infrastructure of the OpenRail Association"?

Reply:
Thanks for the review. Good idea, I changed it like you proposed.